jina/proto/jina_pb2.py

```diff
@@ -16,1426 +17,2763 @@
-  ,
-  dependencies=[google_dot_protobuf_dot_timestamp__pb2.DESCRIPTOR,google_dot_protobuf_dot_struct__pb2.DESCRIPTOR,])
-
+    name='jina.proto',
```

Comment:
Don't reformat the file

Reply:
It's automatic, from precommit config. Does it matter?

Reply:
Yes, it means we would always have 4000 line commits when changing this file. Then it will always get the XL label.
I excluded the file from black.

Reply:
It's not excluded in the precommit, if that's what you meant. I added it now

Reply:
okay nice!